src/main/java/by/exadel/internship/service/Service.java

```diff
@@ -0,0 +1,10 @@
+package by.exadel.internship.service;
+
+import by.exadel.internship.dto.formDTO.FormRegisterDTO;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+public interface Service {
```

Comment:
```suggestion
public interface EmailService {
```
So, you can avoid the naming conflict with @org.springframework.stereotype.Service